src/Tasks.js

```diff
@@ -0,0 +1,46 @@
+import Task from './Task.js';
+import './Tasks.css'
+
+
+function Tasks(props) {
+    let tempData;
+    if (props.isHidden){
+      tempData = props.data.filter((a) => !props.completedTaskIds.includes(a.id));
+    } else {
+       tempData = props.data;
+    }
+
+    // gets text from type box to be added to list
+    function handleAdd() {
+        let x = document.getElementById("addItem").value;
```

Comment:
Not how we do things in React. Instead, have an onChange in your input that updates local state.